column/api/manager/run_manager.py

```diff
@@ -0,0 +1,79 @@
+import copy
+import logging
+import threading
+
+import column
+from column.api import backend
+from column.api.objects import State
+from column.plugins.callback import progress
+
+
+LOG = logging.getLogger(__name__)
+
+
+class RunManager(object):
+    """Column api manager class
+
+    The Manager layer is to support additional logic which is needed
+    to run ansible playbook.
+
+    Attributes:
```

Comment:
i am following http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Do you have example what the format should be?